loopy/kernel/array.py

```diff
@@ -652,7 +652,7 @@ class ArrayBase(ImmutableRecord):
 
     def __init__(self, name, dtype=None, shape=None, dim_tags=None, offset=0,
             dim_names=None, strides=None, order=None, for_atomic=False,
-            target=None, alignment=None,
+            target=None, alignment=None, tags=None,
```

Comment:
Could you make sure this attribute is documented, and documented in the right place?

Reply:
Okay, I added a description in the list of attributes.